### PR TITLE
Read Functionality

### PR DESCRIPTION
**Pull request by LionSnarl**
Bug fixed Header.js line 11 changed className to id="jumbotron".
Read Functionality complete.

### CODE REVIEW

**Review by minachuong — APPROVED** [review]
you could add .eslintcache to your .gitignore file so that it's changes don't get staged/committed 

lookin good!

File: src/pages/CatIndex.js [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
 
 class CatIndex extends Component {
     render(){
+        console.log(this.props.cats)
         return(
             <>
                 <h1>Here are all the cats!</h1>
+                { this.props.cats.map((cat, index) => {
+                    return (
+                        //<p key={index}>{ cat.name}</p>
```
Comment (sjproctor) [inline src/pages/CatIndex.js:13]: If you are not using this first p tag, go ahead and delete it

**Review by sjproctor — APPROVED** [review]
Looks good!

File: src/pages/CatIndex.js (hunk 1 above)
Comment (LionSnarl) [reply src/pages/CatIndex.js:13]: Copy!

**Review by ChrisoloRobin — APPROVED** [review]
Looks great. Smart thinking creating the logic before the css

**Pull request merged**